### PR TITLE
[UFC-114] Update nanoflow documentation

### PR DESCRIPTION
**Pull request by daserra**
[x] Add a new item to  "Differences from Microflows"
[x] Remove "Nanoflow Debugging" section

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
@Luyao-Zhang-1 please let me know when you're done with a quick language revieww

File: content/refguide/nanoflows.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ There are five main differences between nanoflows and microflows:
 3. Nanoflows are not run inside a transaction so, if an error occurs in a nanoflow, it will not roll back any previous changes.
 4. Nanoflows and microflows do not provide the same actions. Some actions available in microflows are not available in nanoflows, and vice versa.
 5. Because nanoflows use JavaScript libraries and microflows use Java libraries, there can sometimes be slight differences in the way expressions are executed.
+6. Changes done to the lists in a subnanoflow are not reflected in the original nanoflow
```
Comment (Luyao-Zhang-1) [inline content/refguide/nanoflows.md:54]: We use 'sub-nanoflow' instead.

Comment (Luyao-Zhang-1) [reply content/refguide/nanoflows.md:54]: @Adam-Dupaski, I finished the language review.

**Review by Robinfr — APPROVED** [review]
Looks good to me.

**Comment by Luyao-Zhang-1** [comment]
@daserra Thank you for your contribution! Could you please sign our [Contributor License Agreement](https://cla-assistant.io/mendix/docs?pullRequest=3429) so that we can accept your contribution?

File: content/refguide/nanoflows.md (hunk 1 above)
Comment (Adam-Dupaski) [reply content/refguide/nanoflows.md:54]: @Luyao-Zhang-1 can you commit this change?

**Comment by daserra** [comment]
> e sign our [Contributor License Agreement](https://cla-assistant.io/mendix/docs?pullRequest=3429) so that we can accept your contribution?

Done!

File: content/refguide/nanoflows.md (hunk 1 above)
Comment (Luyao-Zhang-1) [reply content/refguide/nanoflows.md:54]:
```suggestion
6. Changes done to the lists in a sub-nanoflow are not reflected in the original nanoflow.
```

**Review by Luyao-Zhang-1 — APPROVED** [review]
Thank you for your contribution! Your work has improved our doc.

**Review by Adam-Dupaski — APPROVED** [review]

**Pull request merged**